src/games/gameitem.py

```diff
@@ -268,13 +341,17 @@ def _player_table_row(self, player, align):
 
     def _country_icon_fmt(self, player):
         icon_path_fmt = os.path.join("chat", "countries", "{}.png")
-        icon_path = icon_path_fmt.format(player.country.lower())
+        if player.country is None or player.country == '':
```

Comment:
Either the system is quite insanely designed, or at least one of these possibilities shouldn't exist?

I would suggest just enforcing that `country is None` when unknown, since then this if-block can be replaced with `country = (player.country or '__').lower()` or such.